### PR TITLE
Change executable name, fix apiError display, wait for Base feature

### PR DESCRIPTION
**Pull request by dev0tion**
- Change executable name from SidechainD to CirrusD
- Fix apiError display in the Smart Contracts transaction tab
- Wait until Base feature is fully initialized before loading Core
- Bump version to 1.2.0

### CODE REVIEW

File: StratisCore.UI/src/app/app.component.ts [hunk 1]
```diff
@@ -56,8 +58,18 @@ export class AppComponent implements OnInit, OnDestroy {
...
     this.subscription = stream$.subscribe(
       (data: NodeStatus) => {
-        this.loading = false;
-        this.router.navigate(['login']);
+        this.apiConnected = true;
+        this.statusIntervalSubscription = this.apiService.getNodeStatusInterval(true)
+          .subscribe(
+            response =>  {
+              const statusResponse = response.featuresData.filter(x => x.namespace === 'Stratis.Bitcoin.Base.BaseFeature');
+              if (statusResponse[0].state === 'Initialized') {
```
Comment (fenix2222) [inline StratisCore.UI/src/app/app.component.ts]: Do we need to check if `statusResponse.length > 0`?

Comment (fenix2222) [reply StratisCore.UI/src/app/app.component.ts]: Also, maybe `statusResponse[0].state.toLowerCase() === 'initialized'`?

File: StratisCore.UI/src/app/app.component.ts [hunk 2]
```diff
@@ -56,8 +58,18 @@ export class AppComponent implements OnInit, OnDestroy {
...
+                this.loading = false;
+                this.statusIntervalSubscription.unsubscribe();
```
Comment (fenix2222) [inline StratisCore.UI/src/app/app.component.ts]: Do we need this? Can't we just do `this.apiService.getNodeStatusInterval(true).pipe(take(1))` instead?

File: StratisCore.UI/src/app/app.component.ts (hunk 1 above)
Comment (dev0tion) [reply StratisCore.UI/src/app/app.component.ts]: Hmm yes, I will add it to the if statement.

File: StratisCore.UI/src/app/app.component.ts (hunk 2 above)
Comment (dev0tion) [reply StratisCore.UI/src/app/app.component.ts]: Not sure, I'll test.

Comment (dev0tion) [reply StratisCore.UI/src/app/app.component.ts]: Doesn't do the trick, will leave it as is and add unsubscribe at onDestroy.
Thanks!

**Comment by rowandh** [comment]
Hmm, tested the SC functionality and it's still causing issues. Investigating it now.

**Comment by rowandh** [comment]
Appears to be an issue with the SC API endpoint, so it's not related to this PR.

**Pull request merged**